mods/ra/rules/world.yaml

```diff
@@ -4,6 +4,8 @@
 	ActorMap:
 	ScreenMap:
 	MusicPlaylist:
+		StartingMusic: intro
```

Comment:
I had most of the songs, but not intro.  It would then (as expected) pick a random track (not hellmarch) at startup.